packaging/amazon-linux-ami/ecs.service

```diff
@@ -13,7 +13,8 @@
 # limitations under the License.
 
 [Unit]
-Description=ECS Agent
+Description=Amazon Elastic Container Service - container agent
+Documentation=https://aws.amazon.com/documentation/ecs/
```

Comment:
You could change this to:

```
Documentation=https://aws.amazon.com/documentation/ecs/ https://ecs.aws
```

Reply:
That's a fun url! However, I would prefer to have links that direct users to the documentation pages rather than a product overview page. Is there a path on that domain that points to more specific documentation for the agent or init?

Reply:
Not that I know of, but that's why I left the documentation URL first.  `Documentation` supports a space-separated list of URLs.